src/js/experiments/relatedResources.js

```diff
@@ -1,8 +1,8 @@
 $(() => {
-  // https://app.optimizely.com/v2/projects/16812830475/experiments/20638820544/variations
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20872380274/variations
   window.OptimizelyClient.getVariationName({
-    experimentKey: 'dd_docs_related_resources_mvp_test',
-    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test',
+    experimentKey: 'dd_docs_related_resources_mvp_test_rerun',
+    groupExperimentName: 'q4_fy22_docs_disco_experiment_group_test',
```

Comment:
due to some inaccuracies for the experiment viewed data that gets collected, we have to add in a new option moving forward. All experiments now also need to have the `experimentContainer` key. You can look more into it here: https://github.com/circleci/circleci-docs/pull/5921

Reply:
Good catch! Needed to rebase with master :)